.jenkins/job_beam_Release_NightlySnapshot.groovy

```diff
@@ -34,11 +34,11 @@ mavenJob('beam_Release_NightlySnapshot') {
   common_job_properties.setMavenConfig(delegate)
 
   // Set that this is a PostCommit job.
-  // Polls SCM on Feb 31st, i.e. never.
+  // Does not poll SCM.
```

Comment:
Merge these two lines:
`This is a post-commit job that runs once per day, not for every push.`